### ISSUE TITLE
Piece-wise Saving

### ISSUE THREAD

**Issue by daminitandon** [SaturdayNeighborhoodHealthClinic/osler#125]
It is a major issue that the sections cannot be saved piece-wise (per tab). Is there a way to have the write-up broken up in a way that the sections can be saved as they are completed? This can help in case there are internet issues etc.

**Comment by justinrporter** [comment]
Yes. This is something I have been thinking about for a while. I think that it's high time we did this!

@nakul2234 could you look at [Django formtools](https://github.com/django/django-formtools/), particularly the "form wizard" which I think provides exactly this functionality?

**Comment by nakul2234** [comment]
Yeah, I'll get it done this weekend

**Comment by justinrporter** [comment]
Awesome. I'd like to update on Monday so that we can have this stuff deployed for next week!
